walkthroughs/howto-http-headers/README.md

```diff
@@ -0,0 +1,82 @@
+* 8 Virtual Nodes:
+  1. `front-node`: Sends egress traffic to the `color.header-mesh.local` Virtual Service. It discovers instances via Cloud Map in the `front` service under the `header-mesh.local` namespace. i.e. the DNS name `front.$STACK_PREFIX-mesh.local`.
+  2. `purple-node`: A Cloud Map Node for `color.header-mesh.local` that discovers instances with the attribute `ECS_TASK_DEFINITION_FAMILY` set to `purple`.
+  3. `yellow-node`: A Cloud Map Node for `color.header-mesh.local` that discovers instances with the attribute `ECS_TASK_DEFINITION_FAMILY` set to `yellow`.
+  4. `blue-node`: A Cloud Map Node for `color.header-mesh.local` that discovers instances with the attribute `ECS_TASK_DEFINITION_FAMILY` set to `blue`.
+  5. `green-node`: A Cloud Map Node for `color.header-mesh.local` that discovers instances with the attribute `ECS_TASK_DEFINITION_FAMILY` set to `green`.
+  6. `red-node`: A Cloud Map Node for `color.header-mesh.local` that discovers instances with the attribute `ECS_TASK_DEFINITION_FAMILY` set to `red`.
+  7. `white-node`: A Cloud Map Node for `color.header-mesh.local` that discovers instances with the attribute `ECS_TASK_DEFINITION_FAMILY` set to `white`.
+  8. `black-node`: A Cloud Map Node for `color.header-mesh.local` that discovers instances with the attribute `ECS_TASK_DEFINITION_FAMILY` set to `black`.
+  
+* 7 Routes: 
+  1. `color-route-purple`: A route with a header match rule to match weather the header name `color_header` _does not_ exist.
+  2. `color-route-yellow`: A route with a header match rule to match weather the header name `color_header` _does_ exist.
+  3. `color-route-blue`: A route with a header match rule to match weather the header `color_header` has a value within the range `[100, 150)` (excludes 150).
+  4. `color-route-green`: A route with a header match rule to match weather the header name `color_header` has a value that matches the regex `redor.*`.
+  5. `color-route-red`: A route with a header match rule to match weather the header name `color_header` has a value that matches the prefix `redorgreen`.
+  6. `color-route-white`: A route with a header match rule to match weather the pseudo header name `:method` has a value that matches `GET`.
+  7. `color-route-black`: A route with a header match rule to match weather the pseudo header name `:scheme` has a value that matches the prefix `https`.
+
+## Setup
+
+1. Your account id:
+    ```
+    export AWS_ACCOUNT_ID=<your_account_id>
+    ```
+2. Your region:
+    ```
+    export AWS_DEFAULT_REGION=<i.e. us-west-2>
+    ```
+    
+3. The latest envoy image, see https://docs.aws.amazon.com/app-mesh/latest/userguide/envoy.html
+   ```
+   export ENVOY_IMAGE=<i.e. 111345817488.dkr.ecr.us-west-2.amazonaws.com/aws-appmesh-envoy:v1.11.1.1-prod>
+   ```
+    
+4. Deploy the resources, save the public endpoint output for later (this will take about 5-10 minutes to complete):
```

Comment:
> save the public endpoint output for later

Could you be explicit that the script will output this when it is done executing? I'd even put this as step five.